examples/aws-rds/postgresql/README.md

```diff
@@ -113,20 +113,34 @@ s3-profile-sph7s   2h
 
 
 # Use correct blueprint name (one of `rds-postgres-dump-bp` or `rds-postgres-snapshot-bp`) you have created earlier
-#
-$ kanctl create actionset --action backup --namespacetargets pgtestrds --config-maps dbconfig=pgtestrds/dbconfig --profile pgtestrds/s3-profile-6hmhn -b <blueprint-name> -n kasten-io
-actionset backup-llfb8 created
+
+cat <<EOF | kubectl apply -f -
+> apiVersion: cr.kanister.io/v1alpha1
+> kind: ActionSet
+> metadata:
+>   name: rds-backup
+>   namespace: kasten-io
+> spec:
+>   actions:
+>   - name: backup
+>     blueprint: <blueprint-name>
+>     object:
+>       apiVersion: v1
+>       name: dbconfig
+>       namespace: pgtestrds
+>       resource: configmaps
+>     profile:
+>       name: s3-profile-sph7s
+>       namespace: pgtestrds
+> EOF
+actionset.cr.kanister.io/rds-backup created
```

Comment:
Approving this, but should we change this to `kanctl `command once we have the PR merged in the kanister.